src/main/java/de/teamlapen/vampirism/entity/EntityVampirism.java

```diff
@@ -326,4 +344,18 @@ protected void updateAITasks() {
             onRandomTick();
         }
     }
+
+    /**
+     * gets a list of action combos and adding a random combo to the available actions of the IFaction entity
+     * 
+     * @param actionstmp
+     *            list of action combos
+     */
+    protected void setAvailableActions(List<IEntityAction[]> actionstmp) {
+        if (actionstmp.size() == 0)
+            return;
+        entityActions = new ArrayList<IEntityAction>();
```

Comment:
Maybe assign a list instance on definition at the top of the class and just clear it here. Therefore, the list is never null. Shouldn't make a big difference though